test/end-to-end/core.sh

```diff
@@ -196,11 +196,32 @@ os::cmd::expect_success "docker pull ${DOCKER_REGISTRY}/custom/cross:namespace-p
 echo "[INFO] Cross namespace pull successful"
 
 # check to make sure an image-pusher can push an image
+os::cmd::expect_success "oc project ${CLUSTER_ADMIN_CONTEXT}"
 os::cmd::expect_success 'oc policy add-role-to-user system:image-pusher pusher'
+os::cmd::expect_success 'oc policy add-role-to-user -n test admin pusher'
```

Comment:
don't make the pusher an admin... we're trying to test with just `system:image-pusher` permissions